src/expandableCalendar/agendaList.tsx

```diff
@@ -86,10 +101,11 @@ class AgendaList extends Component<Props> {
 
   componentDidMount() {
     const {date} = this.props.context;
+    const {scrollToFutureEvents} = this.props;
     if (date !== this._topSection) {
       setTimeout(() => {
-        const sectionIndex = this.getSectionIndex(date);
-        if (sectionIndex) {
+        const sectionIndex = scrollToFutureEvents ? this.getNextSectionIndex(date) : this.getSectionIndex(date);
```

Comment:
We should move these few lines to a method as it repeats on didUpdate

Reply:
I moved it into `scrollToSection`